apps/projects/contracts/Projects.sol

```diff
@@ -91,14 +91,28 @@ contract Projects is AragonApp {
         uint index;
     }
 
+    enum SubmissionStatus { Unreviewed, Accepted, Rejected }  // 0: unreviewed 1: Accepted 2: Rejected
+
+    struct WorkSubmission {
+        SubmissionStatus status;
+        string submissionHash; //IPFS hash of the Pull Request
+        uint256 fulfillmentId; // Standard Bounties Fulfillment ID
+    }
+
     struct GithubIssue {
         bytes32 repo;  // This is the internal repo identifier
         uint256 number; // May be redundant tracking this
         bool hasBounty;
+        bool fulfilled;
         uint256 bountySize;
         uint256 priority;
         address bountyWallet; // Not sure if we'll have a way to "retrieve" this value from status open bounties
         uint standardBountyId;
+        address assignee;
+        address[] applicants;
+        address[] workSubmittors;
```

Comment:
Will we have multiple people submit work or only one?

Reply:
I thought it might be good to keep a log of everyone who's submitted work, otherwise we won't be able to have any state where someone's work is rejected. Maybe it's not necessary to store a rejected submission after the task has been reassigned. 

Users can already reset the state of the submissions by resubmitting new work, so that should suffice. I'll update this as well